doc/release_notes.rst

```diff
@@ -5,17 +5,25 @@ Release Notes
 0.2 series
 ..........
 
+0.2.2 (2021-03-19)
+------------------
+
+* Add checks in SBML conversion, e.g. to catch invalid identifiers and equations (# 118).
```

Comment:
```suggestion
* Add checks in SBML conversion, e.g. to catch invalid identifiers and equations (#118).
```
Not familiar but it looks like it's possible to make these links to the corresponding PRs (and issues) if desired.
Above is a typo fix, below is the typo fix + what should be a working link.
```suggestion
* Add checks in SBML conversion, e.g. to catch invalid identifiers and equations ([#118][p118]).
```

Reply:
Might be that the PR numbers in the current format will be automatically linked on the GitHub releases page anyway, so the second suggestion here would be redundant

Reply:
on github, they will be linked already. on rtd in either case not, but that's ok I think.